packages/flame_svg/dart_test.yaml

```diff
@@ -0,0 +1,2 @@
+tags:
+  golden:
```

Comment:
Is this file really needed? We don't have it for our other goldens

Reply:
No, I removed it.

Reply:
Actually, this warning will be shown when running `flutter test` if I remove the file.
```
Warning: A tag was used that wasn't specified in dart_test.yaml.
  golden was used in the test "render sharply"
```
It is also documented in the `golden_toolkit` package ([here](https://pub.dev/packages/golden_toolkit#:~:text=Add%20a%20%22golden%22%20tag%20to%20your%20project)), so I think we should keep it.

Reply:
Aah, seemingly we don't use `golden_toolkit`, but `testGolden` from `flame_test`. Maybe you could migrate to that so that we are consistent between the packages?

Reply:
I updated the test and removed `golden_toolkit`. Ended up using `flutter_test` because I need to show the Svg widget rendered by `flutter_svg` package as well.